nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/resources/log4j.xml

```diff
@@ -6,10 +6,10 @@
   <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
     <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler" />
     <param name="Target" value="System.out" />
-    <param name="Threshold" value="WARN" />
+    <param name="Threshold" value="TRACE" />
```

Comment:
Warn ?

Reply:
No, I want to keep TRACE to let the logs be displayed in the console for the `WorkQueuing` category when uncommenting it.